src/main/scala/com/snowplowanalytics/indicative/Filters.scala

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright (c) 2018-2019 Snowplow Analytics Ltd. All rights reserved.
+ *
+ * This program is licensed to you under the Apache License Version 2.0,
+ * and you may not use this file except in compliance with the Apache License Version 2.0.
+ * You may obtain a copy of the Apache License Version 2.0 at http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the Apache License Version 2.0 is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the Apache License Version 2.0 for the specific language governing permissions and limitations there under.
+ */
+package com.snowplowanalytics.indicative
+
+object Filters {
+
+  val unusedEvents: String = List(
+    "app_heartbeat",
+    "app_initialized",
+    "app_shutdown",
+    "app_warning",
+    "create_event",
+    "emr_job_failed",
+    "emr_job_started",
+    "emr_job_status",
+    "emr_job_succeeded",
+    "incident",
+    "incident_assign",
+    "incident_notify_of_close",
+    "incident_notify_user",
+    "job_update",
+    "load_failed",
+    "load_succeeded",
+    "page_ping",
+    "s3_notification_event",
+    "send_email",
+    "send_message",
+    "storage_write_failed",
+    "stream_write_failed",
+    "task_update",
+    "wd_access_log"
+  ).mkString(",")
```

Comment:
no need to double the formatting/splitting, just keep it as lists here

Reply:
The splitting is just for when users supply their own value via env var, which will be a string. But that then means the defaults must also be passed in as strings. Does that make sense?

Reply:
I think as is, you're letting the configuration format dictate unrelated aspects of your program: the format in which you store defaults. However, they should be independent because as we see here, you're having to turn a list into a string just to turn it back into a list.

Your input data/configuration processing/validation should be a separate concern.

In this case, you can do:

```scala
  def getConfig[T](envVar: String, default: Option[T], transform: String => T): T =
    sys.env.get(envVar).map(transform).orElse(default).getOrElse(throw new RuntimeException(""))
```